xml/architecture-third_party-third_party_service_integration.xml

```diff
@@ -1126,8 +1126,9 @@ ansible-playbook -i hosts/verb_hosts ardana-upgrade.yml</screen>
    <title>Third-party Content Upgrade</title>
    <para>
     It is also possible to update third-party content independently of a
-    &productname; upgrade, e.g. update the plugin used to patch the &productname; Neutron
-    venv. To update third-party content, the following steps apply:
+    &productname; upgrade, e.g. update the plugin used to patch the
```

Comment:
upgrade. For example, update the plugin*

Reply:
changed